wallet/service/starter.go

```diff
@@ -220,7 +220,7 @@ func (s *Starter) buildAPIV1(ctx context.Context, logger *zap.Logger, networkCfg
 	return servicev1.NewAPI(apiV1Logger, handler, auth, forwarder, policy, networkCfg, spam), nil
 }
 
-func (s *Starter) buildAPIV2(ctx context.Context, logger *zap.Logger, cfg *network.Network, pow api.SpamHandler, closer *vgclose.Closer) (*servicev2.API, error) {
+func (s *Starter) buildAPIV2(ctx context.Context, logger *zap.Logger, cfg *network.Network, pow api.SpamHandler, closer *vgclose.Closer, ttl int64) (*servicev2.API, error) {
```

Comment:
Is there a reason to not have a `uint` ? Can we have negative TTL ?

Reply:
Could (and probably should) cast it earlier on. Flags (I don't think) have a `Uint64Var` option, so the value from there is `int64`

Reply:
Alright, I think we should leverage the type system to make things easier to reason about and avoid silly mistakes.